view/src/main/java/cmm/apps/esmorga/view/eventlist/EventListView.kt

```diff
@@ -222,6 +242,16 @@ fun EventList(events: List<EventListUiModel>, onEventClick: (eventId: String) ->
     }
 }
 
+@Composable
+fun <viewModel : LifecycleObserver> viewModel.observeLifecycleEvents(lifecycle: Lifecycle) {
```

Comment:
I think we should move this extension for a generic class, since it can be reused in other composable screens